src/services/pool/decorators/pool.multicaller.ts

```diff
@@ -113,18 +120,6 @@ export class PoolMulticaller {
       }
     });
 
-    result = await multicaller.execute();
-
-    this.pools.forEach(pool => {
-      multicaller.call({
-        key: `${pool.id}.poolTokens`,
-        address: this.vaultAddress,
-        function: 'getPoolTokens',
-        abi: Vault__factory.abi,
-        params: [pool.id],
-      });
-    });
```

Comment:
This separated multicall request was legacy code from when the old multicaller was used and you couldn't use separate ABIs within the same multicall request.